lib/tokens.js

```diff
@@ -342,6 +346,15 @@ export async function resolve({requester, token}) {
     const {tokenBatch} = await _getBatch({id: batchId});
     const {internalId} = tokenBatch;
 
+    if(levelOfAssurance <= tokenBatch.minAssuranceForResolution) {
```

Comment:
This should be `<` instead of `<=`, no?
(Since an initial registration on the Onboarding site will have `minAssuranceForResolution: 2`. And when it's verified by the clerk, the token resolution request will have: `levelOfAssurance: 2`, and it should pass.)

Reply:
Correct, and updated.